packages/strapi-provider-email-mailgun/lib/index.js

```diff
@@ -36,6 +40,7 @@ module.exports = {
 
     const mailgun = mailgunFactory({
       apiKey: config.mailgun_api_key,
+      host: config.mailgun_api_host,
```

Comment:
If `config.mailgun_api_host` is undefined (right after migrating to this new version) does the sdk use the default version or does it break ?

Reply:
Use default https://github.com/bojand/mailgun-js/blob/master/lib/mailgun.js#L26